katalog/cert-manager/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ bases:
   - webhook
 
 commonLabels:
-  app.kubernetes.io/version: 0.9.0
+  app.kubernetes.io/version: 0.14.1
```

Comment:
uhmmmm i will try again the procedure to be sure what the behavior with the labels

Reply:
After a running one more time the migration, i checked the selector of the cert-manager services an they got upgraded (using kubernetes 1.15). Should i try with a specific setup?

Using kubernetes 1.15 + kubectl 1.16 + kustomize 3 seems to be working.

Reply:
Tested the migration/upgrade path in Kubernetes 1.14 + kubectl 1.16. 
Selector updated, everything working.

Reply:
Recap: we can continue with `commonLabels` or remove it if we want.
I like how it is working right now.

Reply:
In the past, I had some trouble trying to update the service's selector but maybe was related to old Kubernetes version. I think it makes sense to leave the label as-is since it could be useful.